src/pages/Home.js

```diff
@@ -133,31 +137,32 @@ const UIDesigns = () => {
       <UIDesignContainer>
         <UIDesignScreenshots
           src="https://static.wixstatic.com/media/71546a_411ab5220f12428bbe241963dfb4181c~mv2.png/v1/fill/w_361,h_707,al_c,lg_1,q_85,enc_auto/Group%201350.png"
-          alt=""
+          alt="first-screenshot"
```

Comment:
alt text is supposed to be descriptive of the image. "first-image" isnt very descriptive